cluster/gce/config-default.sh

```diff
@@ -78,14 +78,14 @@ ETCD_QUORUM_READ="${ENABLE_ETCD_QUORUM_READ:-false}"
 MASTER_TAG="${INSTANCE_PREFIX}-master"
 NODE_TAG="${INSTANCE_PREFIX}-minion"
 MASTER_IP_RANGE="${MASTER_IP_RANGE:-10.246.0.0/24}"
+NODE_IP_RANGE="${NODE_IP_RANGE:-10.40.0.0/22}"
```

Comment:
I know there's not a lot of comments, but can you comment that this is related to aliases?